pkg/flowaggregator/clickhouseclient/clickhouseclient_test.go

```diff
@@ -367,14 +363,41 @@ func TestParseDatabaseURL(t *testing.T) {
 			url:            "tcp://127.0.0.1:9000?key=value",
 			expectedErrMsg: "invalid ClickHouse database URL",
 		},
+		{
+			url:            "tcp://127.0.0.1:9000",
+			expectedProto:  clickhouse.Native,
+			expectedAddr:   "127.0.0.1:9000",
+			expectedErrMsg: "",
```

Comment:
just omit the field when it's not an error case; it makes test case declarations a bit less verbose

Reply:
Fixed. Thanks!